components/needs_team/needs_team.jsx

```diff
@@ -81,35 +84,104 @@ export default class NeedsTeam extends React.Component {
         this.state = {
             team,
             finishedFetchingChannels: false,
+            attemptingToJoinTeam: !team,
```

Comment:
I can't see the use of `attemptingToJoinTeam`.

Reply:
Good catch. I don't need this anymore.